status_dashboard.py

```diff
@@ -0,0 +1,97 @@
+from retriever import install_sqlite
+from retriever.lib.engine_tools import getmd5
+from retriever import datasets
+from retriever.engines import engine_list
+from tempfile import mkdtemp
+from shutil import rmtree
+import os
+import urllib.request
+import json
+from datetime import datetime
+
+mysql_engine, postgres_engine, sqlite_engine, msaccess_engine, csv_engine, download_engine, json_engine, xml_engine = engine_list
+
+file_location = os.path.dirname(os.path.realpath(__file__))
+
+example_datasets = ['bird-size', 'mammal-masses', 'airports', 'portal']
+precalculated_md5 = {'airports': 'f16a0bb21d347e2ba180cde26f35c87a',
+                     'bird-size': '98dcfdca19d729c90ee1c6db5221b775',
+                     'mammal-masses': '6fec0fc63007a4040d9bbc5cfcd9953e',
+                     'portal': 'aaa2b66a3a77efe416e3555c444f4e2e'}
+
+
+def check_urls_exist(dataset):
+    """Check to see if all resources of a dataset
```

Comment:
`def check_urls_exist(dataset):` should be refactored. 
when I read the name I feel like a url is given and return a value based on the check.

Reply:
@henrykironde What should I rename it to? `check_resources`?